### PR TITLE
Initial importing vector drawable support

### PR DESCRIPTION
**Pull request by alexjlockwood**

### CODE REVIEW

File: app/scripts/xmlloader.js [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { Artwork } from './model';
+import { IdProperty } from './modelproperties';
+import { ModelUtil } from './modelutil';
+
+
+export const XmlLoader = {
```
Comment (romannurik) [inline app/scripts/xmlloader.js]: should this be renamed to something more specific like `VectorDrawableXmlLoader`?

File: app/scripts/xmlloader.js [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+          strokeMiterLimit: node.getAttribute('android:strokeMiterLimit') || 4,
+          trimPathStart: node.getAttribute('android:trimPathStart') || 0,
+          trimPathEnd: node.getAttribute('android:trimPathEnd') || 1,
+          trimPathOffset: node.getAttribute('android:trimPathOffset') || 0,
+        });
+      }
+
+      if (node.childNodes.length) {
+        let layers = Array.from(node.childNodes)
+          .map(child => nodeToLayerData_(child))
```
Comment (romannurik) [inline app/scripts/xmlloader.js]: general formatting rule: continuation lines indent 4 spaces instead of 2

**Review by romannurik — CHANGES_REQUESTED** [review]

File: app/scripts/xmlloader.js (hunk 1 above)
Comment (alexjlockwood) [reply app/scripts/xmlloader.js]: Done.

File: app/scripts/xmlloader.js (hunk 2 above)
Comment (alexjlockwood) [reply app/scripts/xmlloader.js]: Done.

**Pull request merged**